twofactor/saltedkey.go

```diff
@@ -0,0 +1,126 @@
+package twofactor
+
+import (
+	"crypto/ecdsa"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+
+	blind "github.com/arnaucube/go-blindsecp256k1"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+	vocdonicrypto "go.vocdoni.io/dvote/crypto/ethereum"
+	"go.vocdoni.io/dvote/crypto/saltedkey"
+)
+
+const (
+	// PrivKeyHexSize is the hexadecimal length of a private key
+	PrivKeyHexSize = 64
+	// SaltSize is the size of the salt used for derive the new key
+	SaltSize = 20
+)
+
+// SaltedKey is a wrapper around ECDSA and ECDSA Blind that helps signing
+// messages with a known Salt. The Salt is added to the private key curve
+// point in order to derive a new deterministic signing key.
+// The same operation must be perform on the public key side in order to
+// verify the signed messages.
+type SaltedKey struct {
```

Comment:
Assure you understand what the SaltedKey is and how it works (https://developer.vocdoni.io/protocol/census/off-chain-csp#blind-signatures). AFAIK we are not using SaltedKeys here.